src/base.jl

```diff
@@ -104,55 +104,56 @@ function System(; buses=[Bus()],
                 services = nothing,
                 annex = nothing,
                 kwargs...)
-    return System(buses, generators, loads, branches, storage, basepower, forecasts, services, annex; kwargs...)
+    return _System(buses, generators, loads, branches, storage, basepower, forecasts, services, annex; kwargs...)
 end
 
-"""Constructs System from a ps_dict."""
-function System(ps_dict::Dict{String,Any}; kwargs...)
+"""Constructs _System from a ps_dict."""
+function _System(ps_dict::Dict{String,Any}; kwargs...)
     buses, generators, storage, branches, loads, loadZones, shunts, forecasts, services =
         ps_dict2ps_struct(ps_dict)
 
-    return System(buses, generators, loads, branches, storage, ps_dict["baseMVA"],
+    return _System(buses, generators, loads, branches, storage, ps_dict["baseMVA"],
                   forecasts, services, Dict(:LoadZones=>loadZones);
                   kwargs...);
 end
 
-"""Constructs System from a file containing Matpower, PTI, or JSON data."""
-function System(file::String, ts_folder::String; kwargs...)
+"""Constructs _System from a file containing Matpower, PTI, or JSON data."""
+function _System(file::String, ts_folder::String; kwargs...)
     ps_dict = parsestandardfiles(file,ts_folder; kwargs...)
     buses, generators, storage, branches, loads, loadZones, shunts, services =
         ps_dict2ps_struct(ps_dict)
 
-    return System(buses, generators, loads, branches, storage, ps_dict["baseMVA"];
+    return _System(buses, generators, loads, branches, storage, ps_dict["baseMVA"];
                   kwargs...);
 end
 
-# - Assign Forecast to System Struct
+# - Assign Forecast to _System Struct
 
 """
 Args:
-    A System struct
+    A _System struct
     A :Symbol=>Array{ <: Forecast,1} Pair denoting the forecast name and array of device forecasts
 Returns:
-    A System struct with a modified forecasts field
+    A _System struct with a modified forecasts field
 """
-function add_forecast!(sys::System,fc::Pair{Symbol,Array{Forecast,1}})
+function add_forecast!(sys::_System,fc::Pair{Symbol,Array{Forecast,1}})
     sys.forecasts[fc.first] = fc.second
 end
 
-"""A System struct that stores all devices in arrays with concrete types.
+"""
+A _System struct that stores all devices in arrays with concrete types.
```

Comment:
We probably shouldn't refer to _System.  You could copy its docstring here and then change _System to say that it is an internal implementation detail.

Reply:
073620e5840a3e2abc5179f3ac3e7499bc23ae9e resolves